src/main/scala/lasp/hapi/server/HapiServer.scala

```diff
@@ -1,4 +1,5 @@
-package latis.server
+package lasp.hapi
+package server
```

Comment:
Why 2 lines for the package?

Reply:
Line too long? ;-)

Reply:
This gives me access to both the `lasp.hapi` and `lasp.hapi.server` namespaces, which I found useful for some stuff in `lasp.hapi.service`. I figured I'd just be consistent and do that everywhere. (Except in tests, which need to import things from multiple places less frequently.)

Here is [an article by Odersky about the change](https://www.artima.com/scalazine/articles/chained_package_clauses_in_scala.html). In particular, he says:

> Generally, if your project consists of multiple subpackages, it's a good idea to use a first package clause that names the project as a whole and is followed by a second package clause naming the current subpackage. That way, you can refer to other subpackages of your project without the often long prefix that indicates the project.

I thought I'd give it a try.